server/src/main/java/io/spine/server/entity/Migration.java

```diff
@@ -238,6 +239,7 @@ private void warnOnNoSystemEventsPosted() {
         private boolean physicallyRemoveRecord;
```

Comment:
In the `Operation` docs it states that it's the operation done on an entity instance. In fact, we apply the migration to the `tx().builder()` now, right?

I guess it really worth adding some more details on how the migration operation is actually being done while you're in the context now.